quiz.js

```diff
@@ -37,6 +53,61 @@ db.dropDatabase();
 
 // your code begins here
 
+db.houses.update({
+  name: 'House Stark'
+}, {
+  $push: {
+    members: {
+      name: 'Ned Stark'
+    }
+  }
+});
+db.houses.update({
+  name: 'House Stark'
+}, {
+  $push: {
+    members: {
+      name: 'Arya Stark'
+    }
+  }
+});
+db.houses.update({
+  name: 'House Stark'
+}, {
+  $push: {
+    members: {
+      name: 'Sansa Stark'
+    }
+  }
+});
+db.houses.update({
+  name: 'House Targaryen'
+}, {
+  $push: {
+    members: {
+      name: 'Viserys Targaryen'
+    }
+  }
+});
+db.houses.update({
+  name: 'House Targaryen'
+}, {
+  $push: {
+    members: {
+      name: 'Daenerys Targaryen'
+    }
+  }
+});
+db.houses.update({
+  name: 'House Arryn'
+}, {
+  $push: {
+    members: {
+      name: 'Jon Arryn'
+    }
+  }
+});
+
 
 
```

Comment:
All correct! There is a shorter way to do this, though, using Mongo's [$each](http://docs.mongodb.org/manual/reference/operator/update/each/) operator:
```
db.houses( { name : 'House Stark' }, { $push : { members : { $each : [
  { name : 'Ned Stark' },
  { name : 'Arya Stark' },
  { name : 'Sansa Stark' },
] } } } )
```